### PR TITLE
Farm 21 security password

### PR DESCRIPTION
**Pull request by Erriep**

### CODE REVIEW

**Review by AntoineCAU — DISMISSED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

File: src/components/Registration/index.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ function Registration() {
 
   const submitHandler = (e) => {
     e.preventDefault();
-    axios.post('http://localhost:5050/users', users);
+    axios.post('http://localhost:5050/auth/signup', users);
```
Comment (LoicBrassart) [inline src/components/Registration/index.jsx]: Don't use a hardcoded "http://localhost", it won't work in production
Time to use Dotenv !

File: src/components/Registration/index.jsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ function Registration() {
           </div>
           <div className="form-group">
             <input
-              type="password"
+              type="text"
```
Comment (LoicBrassart) [inline src/components/Registration/index.jsx]: **Why** would you want to do that on a registration form ? :fearful:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Evils to Fix

**Review by AntoineCAU — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**